src/dataflow/lockfile_packages.rs

```diff
@@ -40,15 +40,11 @@ impl LockfileResult {
     pub fn find_in_directory<P: AsRef<Path>>(directory: P) -> Self {
         let directory = directory.as_ref();
         if !directory.is_dir() {
-            return LockfileResult::LockfileError(LockfileError::IoError(
-                format!("LockfileResult: Manifest must be a file named `wapm.toml` (directory.is_dir() failed on {})", directory.display()),
-            ));
+            return LockfileResult::NoLockfile;
         }
         let lockfile_path_buf = directory.join(LOCKFILE_NAME);
         if !lockfile_path_buf.is_file() {
-            return LockfileResult::LockfileError(LockfileError::IoError(
-                format!("LockfileResult: Manifest must be a file named `wapm.toml` (lockfile_path_buf.is_file() failed on {})", lockfile_path_buf.display()),
-            ));
+            return LockfileResult::NoLockfile;
```

Comment:
If this changes the flow of the user, we need to add a test to verify the new flow